SwrveConversationSDK/src/main/java/com/swrve/sdk/conversations/ui/ConversationFragment.java

```diff
@@ -221,6 +223,15 @@ private void renderControls(Activity activity) {
         }
     }
 
+    private void applyButtonStyles(ConversationStyle style) {
+        Typeface typeface = getTypeface(style, SwrveConversationConstants.DEFAULT_BUTTON_TYPEFACE);
+        style.setTypeface(typeface);
+
+        if(conversationVersion < 4) {
```

Comment:
We should still render as before, change the text size of the button based on the text_size attribute if present